fvm/src/call_manager/default.rs

```diff
@@ -430,6 +570,13 @@ where
                             "fatal error".to_owned(),
                             Err(ExecutionError::Fatal(err)),
                         ),
+                        // XXX this really is not possible dear compiler, Abort::Return
```

Comment:
if something really isn't reachable, you can tell the compiler with `unreachable!()`
```rust 
match result {
    Ok(_) => unreachable!(),
    Err(e) => e,
}